apps/main/src/loan/hooks/useLoanAppStats.tsx

```diff
@@ -8,9 +8,6 @@ import type { ChainId } from '@/loan/types/loan.types'
 import { formatNumber } from '@ui/utils'
 import { t } from '@ui-kit/lib/i18n'
 
-const hasKeys = <K extends keyof any, V>(obj: Record<K, V> | undefined | null): obj is Record<K, V> =>
```

Comment:
Unrelated to the PR but I did it in an older commit anyway and decided to keep it.